tankersdk_identity/test/test_identity.py

```diff
@@ -77,6 +77,32 @@ def test_public_identity_matches_provisional_identity(test_app: Dict[str, str])
     assert public_identity["public_encryption_key"] == identity["public_encryption_key"]
 
 
+def test_public_identity_matches_sms_provisional_identity(
+    test_app: Dict[str, str]
+) -> None:
+    encoded_identity = tankersdk_identity.create_provisional_identity(
+        test_app["id"], "phone_number", "+611223344"
```

Comment:
Nitpick: I guess you meant "+33611223344"? We also have to patch those to use reserved numbers, but later.

Reply:
True. Will fix in the same PR that changes to reserved numbers.